src/patternfly/components/FormControl/form-control.scss

```diff
@@ -220,6 +284,23 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
     background-size: var(--pf-c-form-control--m-search--BackgroundSize);
   }
 
+  &.pf-m-icon {
+    --pf-c-form-control--PaddingRight: var(--pf-c-form-control--m-icon--PaddingRight);
+
+    background-image: var(--pf-c-form-control--m-icon--BackgroundUrl);
+    background-repeat: no-repeat;
+    background-position: var(--pf-c-form-control--m-icon--BackgroundPositionX) var(--pf-c-form-control--m-icon--BackgroundPositionY);
+    background-size: var(--pf-c-form-control--m-icon--BackgroundSizeX) var(--pf-c-form-control--m-icon--BackgroundSizeY);
+  }
+
+  &.pf-m-calendar {
```

Comment:
should the calendar and clock be nested inside of `pf-m-icon` ? in the documentation it was specified as `.pf-c-form-control.pf-m-icon` ?

Reply:
I was trying to keep the specificity down, but I think you're right. We could add a `pf-m-calendar` that does something else, that doesn't apply to `pf-m-icon`, so it would be good to scope that class.